lib/datasources/sop.ts

```diff
@@ -43,6 +43,20 @@ export function makeSopLoader(_req: Request) {
                     AND worktemplatesoplink IS NOT NULL
             `,
         )
+        .with(
+          "workinstance",
+          () => sql`
+                SELECT
+                    id AS _key,
+                    encode(('workinstance:' || id || ':sop')::bytea, 'base64') AS id,
+                    workinstancesoplink AS sop
+                FROM public.workinstance
+                WHERE
+                    id IN ${sql(ids)}
+                    AND workinstancesoplink IS NOT NULL
```

Comment:
lol i didn't know we even had this...

Reply:
We should probably coalesce this with the worktemplatesoplink. It doesn't like even have a workinstance with a non null soplink right now lmao.

```sql
SELECT
    id AS _key,
    encode(('workinstance:' || id || ':sop')::bytea, 'base64') AS id,
    coalesce(workinstancesoplink, worktemplatesoplink) AS sop
FROM public.workinstance
INNER JOIN public.worktemplate
    ON workinstanceworktemplateid = worktemplateid
WHERE
    id IN ${sql(ids)}
    AND workinstancesoplink IS NOT NULL

```

will do fine for now

Reply:
yeah i actually think this is a deprecated column... i always thought worktemplate owned the sop. could check with keller, but could also just ignore for now